app/assets/scss/application.scss

```diff
@@ -1,7 +1,3 @@
-// Reset styles on core HTML elements that might be injected by eg a user's browser, so that we have a known clean base
-// for DM styles to build on.
-@import "toolkit/shared_scss/_reset.scss";
```

Comment:
No, we are not replacing resets. 

Resets are a bit of an anti-pattern when using design systems and components. 

This is what happens when resets are used:

1. tell the browser to remove any default stylings to all elements on a page. 
2. Tell the browser to style all the components using our classes.

With the design system, all components/elements have class names. The components are self-contained and styles do not bleed out into other components that might use the same html elements. It's more efficient to not reset and just go straight to the second instruction because the browser doesn't have to paint the page as part of the first reset and then repaint it again with all the new stylings.

The resets are also the cause for a lot of the custom classes/css introduced to our frontend applications to add spacing around elements.

Happy to chat about it more in detail if needed. 👍

Reply:
Does this include the `body` element? I see that `_reset` is applying the `@include core-19` (typography, I think?) style there.

Reply:
We shouldn't be stying `body` directly. All components should have their own font sizes set using their respective classes eg. (`<p>` using `govuk-body` and `<a>` using `govuk-links` etc)

Reply:
Cool, as long as that's covered elsewhere 👍